apps/mosaico/src/Mosaico/Frontpage/Models.purs

```diff
@@ -0,0 +1,46 @@
+module Mosaico.Frontpage.Models where
+
+import Prelude
+
+import Data.Array (head)
+import Data.Maybe (Maybe(..))
+import Data.Tuple.Nested ((/\))
+import Effect (Effect)
+import KSF.HtmlRenderer.Models as HtmlRenderer
+import Lettera.Models (ArticleStub)
+import Mosaico.MostReadList as MostReadList
+
+data Hook
+  = AndraLaser (Array ArticleStub) (ArticleStub -> Effect Unit)
+
+toHookRep :: Hook -> HtmlRenderer.HookRep
+toHookRep (AndraLaser articles onClickHandler) = andraLaserHook { articles, onClickHandler }
+
+andraLaserHook 
+  :: { articles :: Array ArticleStub
+     , onClickHandler :: ArticleStub -> Effect Unit 
+     } 
+     -> HtmlRenderer.HookRep
+andraLaserHook { articles, onClickHandler } = HtmlRenderer.replacingHook
+  { shouldProcessNode: (\n ->
+                          let info = do
+                                name      <- HtmlRenderer.getName n
+                                attribs   <- HtmlRenderer.getAttribs n
+                                className <- attribs.class
+                                children  <- HtmlRenderer.getChildren n
+                                textChild <- head children
+                                text      <- HtmlRenderer.getData textChild
+                                pure $ name /\ className /\ text
```

Comment:
I would rather just use a PureScript record for this instead of nested tuples. `pure {name, className, text}` and then in case `Just {name, className, text}`.